locksmith/__tests__/app.test.ts

```diff
@@ -97,7 +96,7 @@ describe('Setting lock details', () => {
 
     await request(app)
       .post('/lock')
-      .set('Accept', /json/)
+      .set('Accept', 'json')
```

Comment:
No idea why these were RegExps in the first place, but TS caught that as a `TypeError` and I corrected it.